pandas/_libs/lib.pyx

```diff
@@ -1702,6 +1704,23 @@ cpdef bint is_float_array(ndarray values):
     return validator.validate(values)
 
 
+cdef class ComplexValidator(Validator):
```

Comment:
is this code hit? don't need new tests?

Reply:
this is hit by added test case in test_inference